### PR TITLE
Add arch, duplicate issue and sort method question

### PR DESCRIPTION
**Pull request by hk21702**
Modifies the issue bug report template to add a question about the processor architecture, what sort method they are using if applicable, and also asks the user to first check if their issue had already been reported.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -25,12 +25,17 @@ If applicable, add screenshots to help explain your problem.
 
 **Please complete the following information:**
 
-- OS: [e.g. iOS]
+- Operating System: [e.g. Windows 11]
```
Comment (LionelColaso) [inline .github/ISSUE_TEMPLATE/bug_report.md:28]: Os Naming Can be such as 
Windows (Version) , Mac (Version) , Linux (Name)(Version)

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -25,12 +25,17 @@ If applicable, add screenshots to help explain your problem.
...
+- Processor Architecture: [Arm/x86/Unknown]
```
Comment (LionelColaso) [inline .github/ISSUE_TEMPLATE/bug_report.md:29]: you could add more if you want 
https://en.wikipedia.org/wiki/Comparison_of_instruction_set_architectures

**Review by LionelColaso — APPROVED** [review]
all good modify and merge if u like

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (hk21702) [reply .github/ISSUE_TEMPLATE/bug_report.md:29]: The only ones we support are Arm and x86 so I think its fine lol. Even though I've worked with MIPs assembly before, I doubt we will ever support RimSort MIPs.

**Pull request merged**